edx_api/user_info/__init__.py

```diff
@@ -36,3 +36,34 @@ def get_user_info(self):
         resp.raise_for_status()
 
         return Info(resp.json())
+
+    def update_user_name(self, username, user_full_name):
+        """
+        Updates Full-Name of the user
+
+        Args:
+            username (user.models.User): Username of the Application user
+            user_full_name: Full name to be replace/updated with old full name
+
+        Returns:
+            UserInfo: Object with user details for whom the name is updated
+        """
+        user_name_data = dict(name=user_full_name)
```

Comment:
Some phrasing and variable name nitpicks
```suggestion
    def update_user_name(self, username, full_name):
        """
        Updates full name of the user

        Args:
            username (user.models.User): Username of the Application user
            user_full_name: Full name that will replace the user's existing full name

        Returns:
            UserInfo: Object representing the edX user
        """
        request_data = dict(name=user_full_name)
```

Reply:
@gsidebo this feedback is incorporated.